README.md

```diff
@@ -46,11 +46,18 @@ Then open a web browser to `http://localhost:4000`.
 
 ### Environment
 
-There are a couple settings you may wish to modify in the `./src/assets/environment.json` file as the website may not work otherwise.
-
-- `apiRoot`: Modifying this value will change which api the website will interact with. Currently this defaults to the staging version of the ecosounds api
-- `siteRoot`: Modifying this value will change what the website thinks its current domain is. Currently this is just a redirect to `localhost:4200`
-- `siteDir`: Modifying this value will change what the website thinks its current directory inside the current domain is
+This website can be customised through the environment file located at `./src/assets/environment.json` (note: look at the docker section for deploying environment files). Here is a list of settings and some example values:
+
+Parameter | Description | Example Value
+----------|-------------|--------------
+`build` | Tracks what type of build is being run | `development`<br>`staging`<br>`production`
+`apiRoot` | Any API requests made from the app will use this value for its base and append routes to it | `https://ecosounds.org`<br>`https://ecosounds.org:3000`<br>`https://ecosounds.org/api`
+`clientOrigin` | This is the [origin](https://html.spec.whatwg.org/multipage/origin.html#concept-origin) of where the website will be hosted | `https://ecosounds.org`<br>`https://ecosounds.org:3000`
+`clientDir` | This is the directory on the `clientOrigin` which the website is hosted | `/website`<br>`/web/angular`
+`keys` | This contains the various keys required for third party libraries to run. Check the website for each individual key to understand setup | `googleMaps`: Google maps API key<br>`googleAnalytics`: Google analytics domain and tracking ID
+`brand` | This changes the branding of the website | `short: "Ecosounds",`<br>`long: "Ecosounds - Acoustic Workbench"`
+`links` | This is a list of external links used throughout the website. Check the template for the list of modifiable links |
+`customMenu` | This is a list of custom menu items which changes the contents of the header with instance specific links. Check the template for examples |
```

Comment:
great table, but names of variables gives the impression of a flat structure. Maybe list each in dot notation? E.g. `settings.links`